includes/class-linkbacks-mf2-handler.php

```diff
@@ -139,13 +139,17 @@ public static function generate_commentdata( $commentdata ) {
 		} else { // Fallback to comment author url.
 			$source = $commentdata['comment_author_url'];
 		}
-		if ( ! class_exists( 'Mf2\Parser' ) ) {
-			require_once plugin_dir_path( __DIR__ ) . 'vendor/mf2/mf2/Mf2/Parser.php';
-		}
 
-		// parse source html
-		$parser   = new Parser( $commentdata['remote_source_original'], $source );
-		$mf_array = $parser->parse( true );
+		if ( ! array_key_exists( 'remote_source_mf2', $commentdata ) ) {
+			if ( ! class_exists( 'Mf2\Parser' ) ) {
+				require_once plugin_dir_path( __DIR__ ) . 'vendor/mf2/mf2/Mf2/Parser.php';
+			}
+
+			// parse source html
+			$parser                           = new Parser( $commentdata['remote_source_original'], $source );
```

Comment:
I think this does not work any more, because Webmention is using a different namespace.

Reply:
You are right, forgot about that

Reply:
But this is Semantic Linkbacks... which still bundles the code